### PR TITLE
Update SNAP values to 2025 (effective 2024-10-01)

### LINKED ISSUE #4926: Update SNAP values to 2025 (effective 2024-10-01)
https://www.fns.usda.gov/snap/fy-2025-cola

### PR DESCRIPTION
**Pull request by shuyu7zhan**
Fixes #4926

### CODE REVIEW

**Comment by shuyu7zhan** [comment]
File (https://www.fns.usda.gov/snap/fy-2025-cola) & previous model not align, need to discuss later:
1. page 2, income eligibility standards vs. usda>snap>income>limit>gross/net: rate or absolute #?
2. page 4, min. snap allotments vs. usda>snap>min_allotment>rate: rate or absolute #?
3. page 6, threshold for reporting: not found in model

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Update SNAP values to FY2025.
```
Comment (vrathi101) [inline changelog_entry.yaml:4]:
```suggestion
    - Update SNAP values for 2025.
```

File: policyengine_us/parameters/gov/usda/snap/asset_test/limit.yaml [hunk 2]
```diff
@@ -40,6 +45,12 @@ elderly_disabled:
           href: https://fns-prod.azureedge.us/sites/default/files/resource-files/FY23-Maximum-Allotments-Deductions.pdf
         - title: SNAP - Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2024 Income Eligibility Standards
           href: https://fns-prod.azureedge.us/sites/default/files/resource-files/FY24-SNAP-COLA-Memo.pdfpage=1
+    2024-10-01:
+      value: 4_500
+      reference:
+        - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments
```
Comment (vrathi101) [inline policyengine_us/parameters/gov/usda/snap/asset_test/limit.yaml:51]:
```suggestion
        - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments, Supplemental Nutrition Assistance Program Maximum Asset Limits
```

File: policyengine_us/parameters/gov/usda/snap/income/deductions/excess_shelter_expense/cap.yaml [hunk 3]
```diff
@@ -94,6 +101,8 @@ metadata:
       href: https://www.fns.usda.gov/snap/allotment/COLA
     - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2024 Income Eligibility Standards
       href: https://fns-prod.azureedge.us/sites/default/files/media/file/FY24-Maximum-Allotments-and-Deductions.pdf
+    - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2025 Maximum Allotments and Deductions
```
Comment (vrathi101) [inline policyengine_us/parameters/gov/usda/snap/income/deductions/excess_shelter_expense/cap.yaml:104]:
```suggestion
    - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments, Supplemental Nutrition Assistance Program Deductions, Maximum Excess Shelter Deductions
```

File: policyengine_us/parameters/gov/usda/snap/income/deductions/excess_shelter_expense/homeless/deduction.yaml [hunk 4]
```diff
@@ -30,7 +30,13 @@ values:
       reference:
         - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2024 Income Eligibility Standards
           href: https://www.fns.usda.gov/snap/fy-2024-cola#:~:text=The%20shelter%20cap%20value%20will,and%20the%20U.S.%20Virgins%20Islands.
-    
+  2024-10-01:
+    value: 190.30
+    metadata:
+      reference:
+        - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2025 Income Eligibility Standards
```
Comment (vrathi101) [inline policyengine_us/parameters/gov/usda/snap/income/deductions/excess_shelter_expense/homeless/deduction.yaml:37]:
```suggestion
        - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments, Supplemental Nutrition Assistance Program Deductions, Maximum Homeless Shelter Deductions
```

File: policyengine_us/parameters/gov/usda/snap/income/deductions/standard.yaml [hunk 5]
```diff
@@ -172,6 +202,8 @@ metadata:
       href: https://www.fns.usda.gov/snap/allotment/COLA
     - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2024 Income Eligibility Standards
       href: https://fns-prod.azureedge.us/sites/default/files/media/file/FY24-Maximum-Allotments-and-Deductions.pdf
+    - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2025 Maximum Allotments and Deductions
```
Comment (vrathi101) [inline policyengine_us/parameters/gov/usda/snap/income/deductions/standard.yaml:205]:
```suggestion
    - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments, Supplemental Nutrition Assistance Program Deductions, Standard Deductions
```

File: policyengine_us/parameters/gov/usda/snap/max_allotment.yaml [hunk 6]
```diff
@@ -419,6 +482,9 @@ metadata:
     # Effective 2023-10-01.
     - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2024 Income Eligibility Standards
       href: https://fns-prod.azureedge.us/sites/default/files/media/file/FY24-Maximum-Allotments-and-Deductions.pdf
+    # Effective 2024-10-01.
+    - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2025 Income Eligibility Standards
```
Comment (vrathi101) [inline policyengine_us/parameters/gov/usda/snap/max_allotment.yaml:486]:
```suggestion
    - title: SNAP - Fiscal Year 2025 Cost-of-Living Adjustments, Supplemental Nutrition Assistance Program Maximum Monthly Allotments
```

**Review by vrathi101 — CHANGES_REQUESTED** [review]

**Comment by MaxGhenis** [comment]
This all looks correct to me - can we merge?

File: changelog_entry.yaml (hunk 1 above)
Comment (MaxGhenis) [reply changelog_entry.yaml]: Fine as is but FY2025 is more accurate, since it starts in October 2024.

**Review by MaxGhenis — APPROVED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**